cmd/mattermost/commands/team.go

```diff
@@ -294,6 +303,30 @@ func searchTeamCmdF(command *cobra.Command, args []string) error {
 	return nil
 }
 
+// Restores archived teams by name
+func restoreTeamsCmdF(command *cobra.Command, args []string) error {
+	a, err := InitDBCommandContextCobra(command)
+	if err != nil {
+		return err
+	}
+	defer a.Shutdown()
+	if len(args) < 1 {
```

Comment:
Not a stopper, but we can use `cobra.MinumunNArgs(1)` in the command definition and remove this check.